routes/utils/request-to-bearer-token.js

```diff
@@ -1,8 +0,0 @@
-module.exports = toBearerToken
-
-function toBearerToken (request) {
-  if (!request.headers.authorization) {
-    return ''
-  }
-  return request.headers.authorization.substr(7)
```

Comment:
This makes me wonder... should the prefix for this header be set in the config? Means the code for both `server` and `client` could be automated to some extent as the config would be common across both.